### DISCUSSION TITLE
乾坤3.0的开发计划目前是否仍在进行?

### DISCUSSION THREAD

**Discussion by gmiloveyou** [umijs/qiankun#3025] [Q&A]
想请问乾坤3.0的开发计划目前是否仍在进行中？公司有意向导入使用，但看到状态似乎已有半年没有更新了，担心3.0可能还需要一段时间。可能需要考虑是否使用相对稳定的2.0版本。

**Comment by shanyijin2** [comment]
不在了，噶半路了

**Comment by Mustang-Galaxy** [comment]
看起来是没有投入了

**Comment by littlepoolshark** [comment]
我是 2026 年 3 月 25 来打卡的，看来 3.0 没戏了

**Reply by gmiloveyou** [reply]
哥们儿好，我也来打个卡。离职了，前公司的东西正好用不着
我看原作者今年还有更新记录，感觉之后还有戏。有需要的可以先蹲一下

**Reply by maYunLaoXi** [reply]
vite@2的时候用的，那里候不支持就改用webpack了。现在vite@8了,qiankun@3还没有出(虽然rc也能用），阿里是不是项目裁员了啊

**Comment by haitaodesign** [comment]
前端部都没了，洗洗睡吧🤦‍♂️

码云老细 ***@***.***>于2026年8月27日 周四14:01写道：

**Reply by maYunLaoXi** [reply]
已经转产品+设计+前端了，虽然拿的还是前端的薪水
